data/ships.txt

```diff
@@ -3529,6 +3672,62 @@ ship "Raven"
 
 
 
+ship "Roost"
+	sprite "ship/roost"
+	attributes
+		category "Medium Warship"
+		"cost" 3000000
+		"shields" 2900
+		"hull" 5200
+		"required crew" 7
+		"bunks" 16
+		"mass" 360
+		"drag" 11.5
+		"heat dissipation" .67
+		"fuel capacity" 600
+		"cargo space" 80
+		"outfit space" 450
+		"weapon capacity" 140
+		"engine capacity" 80
+		weapon
+			"blast radius" 80
+			"shield damage" 800
+			"hull damage" 400
+			"hit force" 1200
+	outfits
+		"Meteor Missile Launcher" 2
+		"Meteor Missile" 70
+		"Quad Blaster Turret" 2
+		"Heavy Anti-Missile Turret" 2
+		
+		"S3 Thermionic"
+		"LP072a Battery Pack"
+		"D41-HY Shield Generator" 4
+		
+		"Greyhound Plasma Thruster"
+		"Impala Plasma Steering"
+		"Hyperdrive"
+		
+	engine -45 159
+	engine 45 159
+	gun -15 -128
+	gun 15 -128
+	turret -26 -85 "Quad Blaster Turret"
+	turret 26 -85 "Heavy Anti-Missile Turret"
+	turret -18 114 "Heavy Anti-Missile Turret"
+	turret 18 114 "Quad Blaster Turret"
+	fighter -47 -24 under
+	fighter 47 -24 under
+	fighter -47 50 under
+	fighter 47 50 under
+	explode "tiny explosion" 10
+	explode "small explosion" 25
+	explode "medium explosion" 30
+	explode "big explosion" 15
+	description `Centuries ago, various Hauler models made up nearly half of the merchant vessels in service.  Southbound Shipyards new fighter-carrying containers have created a new use for what many considered an outdated design.  Four external fighter bays make this the biggest civilian carrier on the market, if a little awkward as a warship.  After Southbound assembly workers successfully named the "Nest", "Hauler II: Type F" was quickly dubbed "Roost" to continue the theme.`
```

Comment:
I think this would make more sense.

After Southbound assembly workers successfully renamed the "Hauler I: Type F",